docs/faq/help.txt

```diff
@@ -1,3 +1,5 @@
+.. _faq-getting-help:
```

Comment:
You can use `` :doc:`Getting Help section of the FAQ <faq/help>` `` if you want to refer to the entire page in docs.